src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/XmlObjectSerializerReadContextComplex.cs

```diff
@@ -26,12 +26,14 @@ internal class XmlObjectSerializerReadContextComplex : XmlObjectSerializerReadCo
 
         private bool _preserveObjectReferences;
         private SerializationMode _mode;
+        protected ISerializationSurrogateProvider SerializationSurrogateProvider;
 
```

Comment:
For now it doesn't need to be protected. I was thinking of adding the surrogate support for DCJS as well which would need this to be protected but we've agreed that can be done in a separated  change.